services/metadactyl-clj/src/metadactyl/user.clj

```diff
@@ -17,9 +15,9 @@
   (log/debug user-attributes)
   (let [uid (user-attributes :user)]
     (if (empty? uid)
-      (throw+ {:error_code common-errors/ERR_NOT_AUTHORIZED,
-               :user (dissoc user-attributes :password),
-               :message "Invalid user credentials provided."}))
+      (throw+ {:type :clojure-commons.exception/bad-request-field
```

Comment:
Changed it to correspond to a 401. 
Also fixed the handler to use the proper function (the one which produces 401)